tests/v0_tests/test_marqo_cloud_instance_mapping.py

```diff
@@ -517,25 +517,6 @@ def test_transitioning_flow_without_modifying(self, mock_get):
         with self.assertRaises(MarqoCloudIndexNotFoundError):
             mapping.get_index_base_url("index1")
 
-    def test_index_instantiation_does_not_refresh_urls(self):
```

Comment:
1. Why was this deleted?
2. Docstring needed. For example `This test ensures that when an index object is instantiated, it doesn't cause a refresh of the instance_mapping's urls_mappings, using latest_index_mappings_refresh_timestamp as a proxy indicator`